docs/core/development/policy/code-dev.rst

```diff
@@ -233,7 +218,7 @@ Committing and Post-commit Hooks
 
 
 
-Twisted uses a customized `trac-post-commit-hook <http://bazaar.launchpad.net/~exarkun/twisted-trac-integration/trunk/annotate/head%3A/trac-hooks/trac-post-commit-hook>`_ to enable ticket updates based on svn commit
+Twisted uses a customized `trac-post-commit-hook <http://bazaar.launchpad.net/~exarkun/twisted-trac-integration/trunk/annotate/head%3A/trac-hooks/trac-post-commit-hook>`_ to enable ticket updates based on git commit
```

Comment:
This should be changed to say "Twisted's Trac installation is notified when the Git repository changes, and will update the ticket depending on the Git commit logs." (We don't use trac-post-commit-hook anymore)